packages/visualizations/src/components/Table/Header.svelte

```diff
@@ -13,3 +13,16 @@
         {/each}
     </tr>
 </thead>
+
+<style lang="scss">
+    :global(.ods-dataviz-sdk-table--default) {
+        :global(thead) {
+            border-bottom: 1px solid var(--border-color);
+        }
+        :global(th) {
+            padding: var(--spacing-75);
+            font-weight: normal;
+            text-align: left;
+        }
+    }
```

Comment:
You can split and not use scss nesting if you don't want to add `:global()` on `th, td`. I'm ok with it personnaly 🤷‍♂️